<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="2" passed="0" failed="1" skipped="1">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-11-30T15:42:12 IST" name="Default suite" finished-at="2023-11-30T15:42:18 IST" duration-ms="6852">
    <groups>
    </groups>
    <test started-at="2023-11-30T15:42:12 IST" name="Default test" finished-at="2023-11-30T15:42:18 IST" duration-ms="6852">
      <class name="com.staffModule.ApproveAndViewActiveCustomerPOMTest">
        <test-method signature="generateApplNo()[pri:0, instance:com.staffModule.ApproveAndViewActiveCustomerPOMTest@689604d9]" started-at="2023-11-30T15:42:12 IST" name="generateApplNo" finished-at="2023-11-30T15:42:18 IST" duration-ms="6761" status="FAIL">
          <exception class="java.lang.NullPointerException">
            <message>
              <![CDATA[Cannot invoke "org.apache.poi.ss.usermodel.Cell.getStringCellValue()" because the return value of "org.apache.poi.ss.usermodel.Row.getCell(int)" is null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException: Cannot invoke "org.apache.poi.ss.usermodel.Cell.getStringCellValue()" because the return value of "org.apache.poi.ss.usermodel.Row.getCell(int)" is null
at com.onlinebanking.kredvista.GenericUtils.ExcelUtil.readMultipleData(ExcelUtil.java:101)
at com.staffModule.ApproveAndViewActiveCustomerPOMTest.generateApplNo(ApproveAndViewActiveCustomerPOMTest.java:117)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- generateApplNo -->
        <test-method signature="approveAndView()[pri:0, instance:com.staffModule.ApproveAndViewActiveCustomerPOMTest@689604d9]" started-at="2023-11-30T15:42:18 IST" name="approveAndView" finished-at="2023-11-30T15:42:18 IST" depends-on-methods="com.staffModule.ApproveAndViewActiveCustomerPOMTest.generateApplNo" duration-ms="1" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method com.staffModule.ApproveAndViewActiveCustomerPOMTest.approveAndView() on instance com.staffModule.ApproveAndViewActiveCustomerPOMTest@689604d9 depends on not successfully finished methods 
[com.staffModule.ApproveAndViewActiveCustomerPOMTest.generateApplNo() on instance com.staffModule.ApproveAndViewActiveCustomerPOMTest@689604d9]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method com.staffModule.ApproveAndViewActiveCustomerPOMTest.approveAndView() on instance com.staffModule.ApproveAndViewActiveCustomerPOMTest@689604d9 depends on not successfully finished methods 
[com.staffModule.ApproveAndViewActiveCustomerPOMTest.generateApplNo() on instance com.staffModule.ApproveAndViewActiveCustomerPOMTest@689604d9]
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:150)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- approveAndView -->
      </class> <!-- com.staffModule.ApproveAndViewActiveCustomerPOMTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
